src/native/eventpipe/ep-string.c

```diff
@@ -133,6 +134,36 @@ ep_rt_utf16le_to_utf8_string_n (
 	return ep_utf16_to_utf8_string_impl(str, len, /*treat_as_le*/ true);
 }
 
+ep_char8_t *
+ep_rt_utf8_string_printf_alloc (const ep_char8_t *format, ...)
+{
+    if (!format)
+        return NULL;
+
+    va_list args;
+    va_start (args, format);
```

Comment:
you should do va_copy to rmeove need doing va_start twice.

Reply:
Added another va_list to copy args into.